dist/images/Makefile

```diff
@@ -33,20 +34,26 @@ endif
 	# ${OCI_BIN} push docker.io/ovnkube/ovn-daemonset-ubuntu:latest
 	./daemonset.sh --image=docker.io/ovnkube/ovn-daemonset-ubuntu:latest
 
-fedora: bld
-	${OCI_BIN} build -t ovn-kube-fedora -f Dockerfile.fedora .
+fedora-image: bld
+	OVN_FROM=koji; if [ "${OVN_REPO}" != "" -o "${OVN_GITREF}" != "" ]; then OVN_FROM=source; fi;	\
+	ARGS=""; 																						\
+	if [ "${OVN_REPO}" != "" ]; then ARGS+=" --build-args OVN_REPO=${OVN_REPO}; fi; 				\
+	if [ "${OVN_GITREF}" != "" ]; then ARGS+=" --build-args OVN_GITREF=${OVN_GITREF}; fi; 			\
```

Comment:
So at the end, you are not resolving here OVN_GITREF to a SHA, so we lost those auto-build capabilities that you mentioned?

Reply:
Forgot about that, what do you think about keeping the Dockerfile simple and just do for both OVS and OVN
```
   git init && \                                                                                                     
    git remote add origin $OVN_REPO && \                                                                              
    git fetch origin $OVN_GITREF --depth 1 && \
    git reset --hard FETCH_HEAD && \          
```

And convert at the Makefile the OVN_GITREF into latest SHA?  so it force rebuild if needed.